main.py

```diff
@@ -2,6 +2,38 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    lists_genre = ["Western", "Action", "Dramma"]
+    lists_actor = [
+        ("George", "Klooney"),
```

Comment:
The actor's name 'Klooney' is misspelled. It should be 'Clooney'. This is corrected later in the code, so it might be intentional for demonstration purposes.